### PR TITLE
Add some detail to the status API

### PR DESCRIPTION
**Pull request by lorcan**
Added some details to the status API and some additional formatting.

You can see a preview here: https://github.com/PiSupply/PiJuice/pull/635/files?short_path=7205135#diff-72051353005df7b6afa7f392e91c2c873b7592daac15dd53635ee12dbb3fb881

### CODE REVIEW

File: Software/README.md [hunk 1]
```diff
@@ -987,155 +987,192 @@ Where error_staus value can be NO_ERROR in case data are exchanged with no commu
...
 
 #### PiJuice Status
+
 **GetStatus()**
 
 Gets basic PiJuice status information about power inputs, battery and events.
 Returns:
-'data':{
+```python
+{'data':{
```
Comment (lorcan) [inline Software/README.md:996]: Made some small improvement to some of the returns - added the wrapping curly brackets.

File: Software/README.md [hunk 2]
```diff
@@ -987,155 +987,192 @@ Where error_staus value can be NO_ERROR in case data are exchanged with no commu
...
-}
+}}
+```
 Where:
-* is_fault is True if there faults or fault events waiting to be read or False if there is no faults and no fault events.
-* is_button is True if there are button events, False if not.
-* battery_status is string constant that describes current battery status, one of four: 'NORMAL', 'CHARGING_FROM_IN', 'CHARGING_FROM_5V_IO', 'NOT_PRESENT'.
-* power_input_status is string constant that describes current status of USB Micro power input, one of four: 'NOT_PRESENT', 'BAD', 'WEAK', 'PRESENT'.
-* 5v_power_input_status: is string constant that describes current status of 5V GPIO power input, one of four: 'NOT_PRESENT', 'BAD', 'WEAK', 'PRESENT'.
+* `is_fault` is `True` if there are faults or fault events waiting to be read or `False` if there are no faults and no fault events.
```
Comment (lorcan) [inline Software/README.md:1005]: Added some code formatting for variables.

File: Software/README.md [hunk 3]
```diff
@@ -987,155 +987,192 @@ Where error_staus value can be NO_ERROR in case data are exchanged with no commu
...
+```python
+{'data': 57, 'error': 'NO_ERROR'}
+```
 
 **GetFaultStatus()**
-Get current fault status of PiJuice HAT.
+
+Get the current fault status of PiJuice HAT.
+
+Example:
```
Comment (lorcan) [inline Software/README.md:1042]: Added an example to bring this in line with the other API documentation.

File: Software/README.md [hunk 4]
```diff
@@ -987,155 +987,192 @@ Where error_staus value can be NO_ERROR in case data are exchanged with no commu
...
 ```python
 {'data': {'button_power_off': True, 'forced_power_off': True}, 'error': 'NO_ERROR'}
 ```
-Where data equals:
-button_power_off
-forced_power_off
-forced_sys_power_off
-watchdog_reset
-battery_profile_invalid ['NORMAL', 'SUSPEND', 'COOL', 'WARM']
+Where data contains:
```
Comment (lorcan) [inline Software/README.md:1050]: I think this is better but it's not fully documented.

File: Software/README.md [hunk 5]
```diff
@@ -1152,106 +1189,140 @@ sleep(1.25)
...
+```
 
 **SetIoDigitalOutput(pin, value)**
+
 Sets state at IO pin configured as digital output.
+
 Arguments:
-pin: IO pin designator, 1 for IO1, 2 for IO2.
-value: output state to set, 0 for low output state, 1 for high.
+* `pin`: IO pin designator, 1 for IO1, 2 for IO2.
```
Comment (lorcan) [inline Software/README.md:1243]: Separated arguments into bullet-lists.

File: Software/README.md [hunk 6]
```diff
@@ -1152,106 +1189,140 @@ sleep(1.25)
...
-where duty_circle is pulse width as percentage of period.
+where `duty_circle` is pulse width as percentage of period.
+
 Example:
 ```python
 print(pijuice.status.GetIoPWM(2))
 ```
 Returns:
 ```python
-`{'data': 35.59984130375072, 'error': 'NO_ERROR'}`
```
Comment (lorcan) [inline Software/README.md:1254]: Removed superfluous curly brackets.

**Comment by shawaj** [comment]
@lorcan thanks very much for this. Looks good. I'm on a phone right now but will probably merge when I get to a machine, unless I find any issues 👌

**Comment by lorcan** [comment]
Great, thanks @shawaj, let me know if there's any improvements I can make. I'm working through the documentation myself as I learn how to use the PiJuice. Thanks for such an amazing piece of kit!

**Comment by lorcan** [comment]
Hi @shawaj, did you get a chance to look at these changes to the docs? Happy to make any changes required :-)

**Comment by tvoverbeek** [comment]
Looks good. Merged.

**Comment by lorcan** [comment]
Great, thanks folks!

**Pull request merged**